### ISSUE TITLE
a small question about Deepglobe dataset

### ISSUE THREAD

**Issue by Jack-bo1220** [VinAIResearch/MagNet#23]
Hi,
First of all, thank you for your excellent work. However, I have a small question about Deepglobe dataset.

1. As far as I know, the original validation and test sets do not provide labels. Are all the val and test sets in the current data split from the original training set?
2. According to your and GLNet's split file (.txt), the total number of train samples is 454 rather than 455. Why do you and your previous papers on related work explain that 455？

Thank you for your attention!

**Comment by hmchuong** [comment]
1. Yes, they are extracted from the training sets.
2. I just checked and it's 454, there may be a typo
